.cron.yml

```diff
@@ -32,3 +32,9 @@ jobs:
           treeherder-symbol: bump-ac
           target-tasks-method: bump_android_components
       when: [{hour: 14, minute: 0}]
+    - name: ui-test-screenshots-x86
+      job:
+          type: decision-task
+          treeherder-symbol: ui-test-screenshots-x86-D
+          target-tasks-method: ui-test-screenshots-x86
+      when: [{hour: 4, minute: 0}]
```

Comment:
I guess this is to run the task daily at that time, is it possible to set the run to once a week for example? I don't think is necessary to run them daily...

Reply:
https://fossies.org/linux/firefox/taskcluster/docs/cron.rst

e.g, Mondays and Thursdays at 10 AM

``` 
when:
    - {weekday: 'Monday', hour: 10, minute: 0}
    - {weekday: 'Thursday', hour: 10, minute: 0}